blocked-edges/4.12.54-RHELKernelCephFSCapDropPanic.yaml

```diff
@@ -0,0 +1,15 @@
+to: 4.12.54
+# 4.12 before 4.12.54 and 4.11
+from: 4[.](11[.].*|12[.]([1-4]?[0-9]|5[0123]))[+].*
+url: https://issues.redhat.com/browse/COS-2705
+name: RHELKernelCephFSCapDropPanic
```

Comment:
nit: our only Ceph declaration so far is:

```console
$ git grep -oh 'name: .*Ceph.*' | sort | uniq -c
     13 name: CephParallelFsync
```

We didn't call out `RHELKernel` or `FS` then.  If we follow that pattern here, it would be `CephCapDropPanic`.  But :shrug:, no need to be particularly consistent vs. old risk names, if you prefer the additional length/context.

Reply:
I followed `RHELKernelHighLoadIOWait` but I kinda like shorter names and maybe we dont want to give RHEL bad publicity